indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -1066,9 +1076,25 @@ private void taskComplete(
     return assignedTasks;
   }
 
-  public List<ZkWorker> getLazyWorkers()
+  @Override
+  public Collection<Worker> getLazyWorkers()
   {
-    return ImmutableList.copyOf(lazyWorkers.values());
+    return ImmutableList.copyOf(getWorkerFromZK(lazyWorkers.values()));
+  }
+
+  protected Collection<Worker> getWorkerFromZK(Collection<ZkWorker> workers)
```

Comment:
I'm going to go the other way here actually, and make this a public static, even if it isn't used elsewhere